devel/natscli/Portfile

```diff
@@ -0,0 +1,300 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/nats-io/natscli 0.0.35 v
+revision            0
+categories          devel
+maintainers         {@0x1DA117 danieltrautmann.com:me} openmaintainer
+license             Apache-2
+
+description         The NATS Command Line Interface
+
+long_description    A command line utility to interact with and manage NATS.
+
+build.env-append    CGO_ENABLED=0
+# Allow Go to fetch dependencies at build time.
+# This is necessary because the package github.com/AlecAivazis/survey
+# changed its owner to github.com/go-survey/survey, but the go.mod
+# still specifies the old location.
+build.env-delete    GO111MODULE=off GOPROXY=off
```

Comment:
If you are allowing Go to fetch its own dependencies, then Go isn't really using the dependencies provided by the `go.vendors` section.  You might as well remove that entire block, and have your `checksums` block simply be the checksums without any filenames (no `${distname}${extract.suffix}`).

Reply:
but that is ideally not what we want - right?